client/src/components/Metrics/MetricsEpoch/MetricsEpoch.tsx

```diff
@@ -156,13 +132,6 @@ const MetricsEpoch = (): ReactElement => {
           totalPersonal={totalPersonal}
           totalUserDonations={sumOfDonations}
         />
-        {epoch < 4 && (
-          <MetricsEpochGridBelowThreshold
-            className={styles.belowThreshold}
-            ethBelowThreshold={ethBelowThreshold}
-            isLoading={isLoading}
-          />
-        )}
```

Comment:
We won't show historic ETH below threshold at all?

Reply:
There is no place in new design for that (new row with one tile).